### PR TITLE
Last milestone

### PR DESCRIPTION
**Pull request by joyapisi**
## What I Have Done 🖱💻 :
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,46 +1,138 @@
...
+        // const target = e.target;
+        // target.setAttribute('contenteditable', 'true');
+        // const content = document.querySelector('[contenteditable]');
+        // content.addEventListener('keyup', (e) => {
+        //   e.preventDefault();
+        //   if (e.key === 'Enter') {
+        //     target.textContent = content.textContent;
+        //     target.setAttribute('contenteditable', 'false');
+        //   }
+        // });
```
Comment (muneebulrehman) [inline src/index.js]: Kindly remove the commented out code to keep your code base clean.

**Review by muneebulrehman — APPROVED** [review]
Hi @joyapisi , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- All technical requirements met for this milestone.

## Required Changes ♻️
_Check the comments under the review._

- [x] All the methods related to the status updates (completed: true/false) should be in a new file that is imported as a module. Kindly do so before submitting for a re-review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joyapisi** [comment]
## What I Have Done 🖱💻 :
-add status update function its own javascript file in the modules folder

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @joyapisi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**